core/src/test/java/org/apache/iceberg/rest/responses/TestCreateNamespaceResponse.java

```diff
@@ -165,8 +163,12 @@ public CreateNamespaceResponse createExampleInstance() {
 
   @Override
   public void assertEquals(CreateNamespaceResponse actual, CreateNamespaceResponse expected) {
-    Assert.assertEquals("Namespaces should be equal", actual.namespace(), expected.namespace());
-    Assert.assertEquals("Properties should be equal", actual.properties(), expected.properties());
+    Assertions.assertThat(actual.namespace())
+        .as("Namespaces should be equal")
```

Comment:
I don't think the `.as()` provides any additional help during debugging, so we can omit it here